sklearn/feature_extraction/text.py

```diff
@@ -937,6 +937,10 @@ class TfidfTransformer(BaseEstimator, TransformerMixin):
     sublinear_tf : boolean, default=False
         Apply sublinear tf scaling, i.e. replace tf with 1 + log(tf).
 
+    additional_idf : boolean, default=True
```

Comment:
Also, it should probably be more explicit in the original docstring? And in both the TfidfTransformer and the TfidfVectorizer?